java/ql/src/Telemetry/DatabaseQualityDiagnostics.ql

```diff
@@ -0,0 +1,40 @@
+/**
+ * @name Low Java analysis quality
+ * @description Low Java analysis quality
```

Comment:
```suggestion
 * @name Low-quality Java analysis
 * @description Low-quality Java analysis
```
This is very minor! Not sure if this title is going to be displayed to the user, but I think this might be slightly more direct if so

Reply:
Can't help but chime in here. I prefer the original, since I read slightly different semantics from those two versions, and the query detects whether the "analysis quality" is low - whereas describing the "Java analysis" as "Low-quality" just seems wrong to me.

Reply:
@aschackmull thanks, makes sense! Fair enough, if its inaccurate then definitely feel free to ignore this suggestion! 👍